SensorNode/sensor_nodes/onBoxSensorNode.py

```diff
@@ -56,6 +57,26 @@ def __init__(self, config_path: str):
             # absolute() not necessary because of ASMSensorNode dir
             self.ff_log_dir = pathlib.Path(appdirs.user_log_dir('ASMSensorNode'), 'ffmpeg_logs')
 
+        self.extra_endpoints = sensor_params['extra_endpoints']
+        if self.extra_endpoints is not None:
+            if len(self.extra_endpoints) ==0:
+                self.extra_endpoints = None
+            else:
+                # Check if extra_endpoints a list 
+                try:
+                    assert isinstance(self.extra_endpoints, list)
+                except AssertionError as e:
```

Comment:
Variable name "e" doesn't conform to snake_case naming stylepylint(invalid-name)
Unused variable 'e'pylint(unused-variable)